### ISSUE TITLE
How long can I expect it to take to generate a 7 characters + digit prefix?

### ISSUE THREAD

**Issue by bmintz** [cathugger/mkp224o#3]
I'm currently searching for posters2, 3, 4, 5, 6, and 7. 

With 1.4 MH/s, it's been 4 hours 45 minutes with no hit. Is this out of my reach, or can I expect an output within like a day or so?

**Comment by cathugger** [comment]
some unscientific math incoming...
it took few hours (dont remember exact ammount of time tbh) for me to make 6 char onion with ~415000calc/sec rate..
you've got more power but you want much longer onion too.
every character divides probability to hit 32 times.
so in your case, you want to generate 2 characters more than I did, and last character has 6 variants...
so it'd take (32 * 32 / 6) longer for me to generate such onion on my hardware compared to one I generated before.
now, you've got more hash power, so (32 * 32 / 6) / (1.4 / 0.415) ~= 50.59047619
tl;dr give it few days
